.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,28 @@
+<!--- Provide a general summary of your changes in the title above -->
+
+## Purpose and Motivation
+
+<!--- Please describe the purpose and motivation of the pull request. -->
+<!--- Why is this change required? What problem does it solve? -->
+<!--- If it fixes an open issue, please link to the issue here. -->
+
+## Types of changes
+
+<!--- What types of changes does your pull request introduce? Put an `x` in all the boxes that apply: -->
+- [ ] Documentation (non-code change which corrects or adds documentation for existing features)
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to change)
+
+## Checklist
```

Comment:
The checklist is to fill out over the course of the PR, no? Or what is the intention of having checkboxes to tick?